extensions/wikibase/src/org/openrefine/wikibase/qa/EditInspector.java

```diff
@@ -226,26 +233,50 @@ private void resolveWarningPropertyLabels() throws ExecutionException {
 
             warningStore.getWarnings().stream()
                     .forEach(warning -> {
-                        // Iterate through each entry in the properties map
                         warning.getProperties().forEach((key, value) -> {
-                            // Check if the value is of type PropertyIdValue
                             if (value instanceof PropertyIdValue) {
                                 PropertyIdValue property = (PropertyIdValue) value;
                                 String label = "";
 
-                                // Retrieve the label based on the locale
                                 PropertyDocument propertyDocument = (PropertyDocument) entityCache.get(property);
                                 if (propertyDocument != null && propertyDocument.getLabels() != null) {
                                     label = propertyDocument.getLabels().get(Locale.getDefault().getLanguage()) != null
                                             ? propertyDocument.getLabels().get(Locale.getDefault().getLanguage()).getText()
                                             : "";
                                 }
-
-                                // Update the property with a new SuggestedPropertyIdValue
                                 warning.setProperty(key, new SuggestedPropertyIdValue(property.getId(), property.getSiteIri(), label));
                             }
```

Comment:
Here would be a good place to add handling for `ItemIdValue` in addition to `PropertyIdValue`

Reply:
Done